### PR TITLE
[DOCS] Remove reference to validation operator process from how to trigger slack notifications guide

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

Changes proposed in this pull request:
- Correction to Additional Information (`notify_with` is configured in Checkpoint config, not `great_expectations.yml`)
- Update 2nd tab group to use different ID (so that switching tabs in that group will not also switch tabs in the first group, causing the page to scroll.)
- Minor formatting edits to bring in line with the documentation standards.

After submitting your PR, CI checks will run and @cla-bot will check for your CLA signature.

For a PR with nontrivial changes, we review with both design-centric and code-centric lenses.

In a design review, we aim to ensure that the PR is consistent with our relationship to the open source community, with our software architecture and abstractions, and with our users' needs and expectations. That review often starts well before a PR, for example in github issues or slack, so please link to relevant conversations in notes below to help reviewers understand and approve your PR more quickly (e.g. `closes #123`).

Previous Design Review notes:
-
-
-

### Definition of Done
Please delete options that are not relevant.

- [X] I have made corresponding changes to the documentation

Thank you for submitting!

### CODE REVIEW

File: docs/guides/validation/validation_actions/how_to_trigger_slack_notifications_as_a_validation_action.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ Open the `.yml` configuration file in `great_expectations/checkpoints` that corr
 Add the `send_slack_notification_on_validation_result` Action to the `action_list` section of the configuration. Make sure the following section exists in the Checkpoint configuration.
 
 <Tabs
-  groupId="webhook-or-app"
+  groupId="webhook-or-app2"
```
Comment (talagluck) [inline docs/guides/validation/validation_actions/how_to_trigger_slack_notifications_as_a_validation_action.md:69]: Is this change intentional?

**Review by talagluck — APPROVED** [review]
LGTM! One small question.

File: docs/guides/validation/validation_actions/how_to_trigger_slack_notifications_as_a_validation_action.md (hunk 1 above)
Comment (Rachel-Reverie) [reply docs/guides/validation/validation_actions/how_to_trigger_slack_notifications_as_a_validation_action.md:69]: It is.  Changing the group ID causes the two tab blocks to no longer be linked when a user changes tabs on one of them.  This prevents changing tabs in the lower tab group to cause the text to "jump" due to the earlier tab group also changing tabs and resizing.
